core/src/main/java/com/netease/arctic/table/TableProperties.java

```diff
@@ -69,66 +69,103 @@ private TableProperties() {
   /**
    * table optimize related properties
    */
-  public static final String ENABLE_OPTIMIZE = "optimize.enable";
-  public static final String ENABLE_OPTIMIZE_DEFAULT = "true";
+  public static final String ENABLE_SELF_OPTIMIZING = "self-optimizing.enabled";
```

Comment:
Shall we change `log-store.enable` to `log-store.enabled` and `dim-table.enable` to `dim-table.enabled` too ?

Reply:
Ok, I have changed them, and also the `kerberos.enabled` `kerberos.proxy.enabled` `spark.arctic.sql.delegate.enabled` `metrics.event-latency.enabled` `metrics.enabled` `log-store.consistency-guarantee.enabled` `arctic.ams.ha.enabled` `arctic.enabled`.